src/Native/Unix/configure.cmake

```diff
@@ -174,6 +174,20 @@ check_cxx_source_compiles(
 
 check_cxx_source_compiles(
     "
+    #include <dirent.h>
+    int main(void)
+    {
+        DIR* dir;
+        struct dirent* entry;
+        struct dirent* result;
+        readdir_r(dir, entry, &result);
```

Comment:
Ah cool, if you have already checked then no need...I just forgot what options we used...thanks for already verifying!